packages/xcoobee-cookie-kit-react/src/CookieKitContainer.js

```diff
@@ -159,66 +167,70 @@ export default class CookieKitContainer extends React.PureComponent {
     }
   }
 
-  componentDidMount() {
-    if (this.state.countryCode) {
-      this.getCookieConsents();
-    } else {
-      CookieConsentsManager.fetchCountryCode()
+  onLoginStatusChange(loginStatus) {
+    // console.log("CookieKitContainer#onLoginStatusChange");
+    const { isLoginStatusChecked } = this.state;
+
+    this.setState({ loginStatus });
+
+    if (!isLoginStatusChecked) {
+      this.setState({ isLoginStatusChecked: true });
+      this.getCountryCode()
         .catch((error) => {
```

Comment:
I think this `catch` should be either in `fetchCountryCode` if writing an error to the console and returning `null` is how we want to handle errors when calling `fetchCountryCode` or should be in `getCountryCode`.